content/teknologi/altinnstudio/altinn-api/app-api/data-element/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Data elements
+description: The App API to work with the data elements associated with an instance
+toc: true
+tags: [api]
+weight: 100
+---
+
+## Overview
+
+A Data element consist of two parts; its metadata document and the actual data blob. This API work primarily with the blob while keeping the metadata document updated.
+
+**basePath**
+```http
+{org}/{appname}/instances/{instanceOwnerPartyId}/{instanceGuid}/data
+```
+
+## Get data 
+
+Endpoint for downloading the data blob.
+
+```http
+GET basePath/{dataGuid}
+Accept: application/{xml/json}
```

Comment:
Asking because I don't know.. Will this accept header allow for the return of attachments? 
Or should this information be linked to the form data section?

Reply:
The accept header is used when picking a serializer. This isn't done by us in our code, but by the framework. All we do is call the `Ok` or `Created` method and the framework does the rest. For Attachments we use the `File` method, which I believe ignores the Accept header and instead use the Content-Type given to it as a parameter.